templates/stamp/stamp.json

```diff
@@ -147,11 +147,59 @@
                 "description": "Number of application virtual machines behind the load balancer."
             }
         },
-        "branch": {
+        "customInstallerGithubAccountName": {
+            "type": "string",
+            "defaultValue": "microsoft",
+            "metadata": {
+                "description": "Name of the GitHub account where the custom installer lives"
+            }
+        },
+        "customInstallerGithubProjectName": {
+            "type": "string",
+            "defaultValue": "oxa-tools",
```

Comment:
Should this be the private repo, requiring the access token? i.e., oxa-tools-config